guides/common/modules/proc_configuring-repositories-foreman-deb.adoc

```diff
@@ -0,0 +1,24 @@
+.Procedure
+Select the operating system and version you are installing on:
+
+* xref:#repositories-debian-11[Debian 11 (Bullseye)]
+* xref:#repositories-ubuntu-2004[Ubuntu 20.04 (Focal)]
+* xref:#repositories-ubuntu-2204[Ubuntu 22.04 (Jammy)]
```

Comment:
These custom tables of contents (ToCs) are kind of frowned upon. It's a maintenance burden (What if someone adds another OS? Will they remember to update the list? My experience says that they often don't.) and it duplicates the regular ToC.

However, if the goal is to enable the implementation of tabs, I assume you wouldn't even need to use a list like this. So if this is just one of the steps on the path to https://github.com/theforeman/foreman-documentation/pull/3104, feel free to ignore this comment.

Reply:
> However, if the goal is to enable the implementation of tabs, I assume you wouldn't even need to use a list like this. So if this is just one of the steps on the path to #3104, feel free to ignore this comment.

That is exactly my goal: I want to get rid of them and my tabs PR intends to do that. This PR is just about moving things so it's obvious we're not touching downstream when we do that.